Formula/c/cahute.rb

```diff
@@ -0,0 +1,92 @@
+class Cahute < Formula
+  include Language::Python::Virtualenv
+
+  desc "Library and set of utilities to interact with Casio calculators"
+  homepage "https://cahuteproject.org/"
+  url "https://ftp.cahuteproject.org/releases/cahute-0.3.tar.gz"
+  sha256 "e84fdcbfc901b7818900fd959dab110a8aeb7d17ad8ac2f79287436d2519fdfa"
+  license "CECILL-2.1"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => [:build, :test]
+  depends_on "python-setuptools" => :build
```

Comment:
we should vendor this out and log an ticket of removing setuptools usage. (but that can be followup)